lib/sync/lock.js

```diff
@@ -1,23 +1,31 @@
-// TODO Use mongodb for locking via https://github.com/chilts/mongodb-lock
-// TODO when using https://github.com/chilts/mongodb-lock, ensure old locks are removed
+var mongoLock = require('@akeating-redhat/mongodb-lock');
 
+module.exports = function(mongoClient, collectionName) {
+  collectionName = collectionName || 'sync_locks';
+  return {
+    locks: {},
```

Comment:
very small thing but I don't think the `locks` variable and the `findOrCreate` function need to be public? They could be private functions.